src/bouwdossiers/batch.py

```diff
@@ -98,7 +98,15 @@ def add_wabo_dossier(x_dossier, file_path, import_file, count, total_count):  #
     Add wabo dossier to the the bouwdossier model. Structure of import is
     almost identical to pre_wabo xml to avoid confusion since only a few
     mappings are different and new fields are added.
+
+    Originally this was for importing wabo dossiers. But not it is also
```

Comment:
typo: `not` -> `now`

Reply:
Dit zal ik aanpassen in een volgende PR omdat deze al is gemergd